### ISSUE TITLE
Issues with Unifi Controller 7.5.174 (metric fetch for InfluxDB failed)

### ISSUE THREAD

**Issue by dladrach48** [unpoller/unpoller#605]
Since I upgraded to 7.5.174 I am seeing the below error messages in the logs.

[ERROR] metric fetch for InfluxDB failed: unifi.GetClients(https://192.x.x.x:8443): json: cannot unmarshal number 0.0 into Go struct field Client.data.bytes-r of type int64

[ERROR] metric fetch for InfluxDB failed: unifi.GetClients(https://192.x.x.x:8443): json: cannot unmarshal number 14.859218182528215 into Go struct field Client.data.bytes-r of type int64

[ERROR] metric fetch for InfluxDB failed: unifi.GetClients(https://192.x.x.x:8443): json: cannot unmarshal number 6.111371446422993 into Go struct field Client.data.bytes-r of type int64

**Comment by claytondukes** [comment]
Same problem here - any ideas would be welcome

**Comment by claytondukes** [comment]
I was checking the latest release and noticed that my docker-compose.yml uses `golift/unifi-poller:latest` which, apparently, is no longer used. I tried `ghcr.io/unpoller/unpoller:latest` which now throws different errors:


```
2023/09/06 20:40:43 [INFO] Re-authenticating to UniFi Controller: https://127.0.0.1:8443
2023/09/06 20:40:43 [ERROR] metric fetch for InfluxDB failed: re-authenticating to https://127.0.0.1:8443: unifi controller: making request: Get "https://127.0.0.1:8443/": dial tcp 127.0.0.1:8443: connect: connection refused: error initializing inputs
2023/09/06 20:41:13 [INFO] Re-authenticating to UniFi Controller: https://127.0.0.1:8443
2023/09/06 20:41:13 [ERROR] metric fetch for InfluxDB failed: re-authenticating to https://127.0.0.1:8443: unifi controller: making request: Get "https://127.0.0.1:8443/": dial tcp 127.0.0.1:8443: connect: connection refused: error initializing inputs
```

Not sure if these are related, I'll poke around a bit.

**Comment by dladrach48** [comment]
> Turns out, the new version is just vastly different. I had to start over :(

Ugh, sorry, the tool is awesome.

**Comment by claytondukes** [comment]
Agreed, and worth starting over :)

I've already rebuilt it.

**Comment by platinummonkey** [comment]
Duplicate of #601

**Comment by dladrach48** [comment]
> Agreed, and worth starting over :)
> 
> I've already rebuilt it.

Just grab the latest?

**Comment by claytondukes** [comment]
>Just grab the latest?
I did, but the latest wasn't compatible with the old one.

**Comment by SeekingHelp24** [comment]
So what did y'all do to resolve this? I am confused. I am having the same issue with no data being populated into the InfluxDB and Unpoller itself looking for 127.0.0.1:8443 as the controller. However, the controller and unpoller are not on the same device. This worked for a year and a half without problem, now nothing I have tried has been able to get this back to functional. Is there anyone that could possible give me some assistance? We leveraged this ability with a client, they're asking to see reports at the end of their festival next week regarding network usage, but as of right now I have nothing! I am on the newest 2.9.2 i think it is? for unpoller. 10.0.3 I believe is the Grafana edition and Influxdb 1.8.x
